File: LacesAPI/LacesViewModel/Request/ProcessTransactionRequest.cs
namespace LacesViewModel.Request
{
    public class ProcessTransactionRequest : LacesRequest
    {
        public int BuyerId { get; set; }
        public int SellerId { get; set; }
        public int ProductId { get; set; }
        public string ReferenceNumber { get; set; }
        public double Amount { get; set; }
    }
}

File: JavaHttpClasses/Request/ProcessTransactionRequest.java
public class ProcessTransactionRequest extends ProductRequest
{
	private int BuyerId;
	private int SellerId;
	private String ReferenceNumber;
	private double Amount;
	
	public int getBuyerId()
	{
		return BuyerId;
	}
	
	public int getSellerId()
	{
		return SellerId;
	}
	
	public String getReferenceNumber()
	{
		return ReferenceNumber;
	}
	
	public double getAmount()
	{
		return Amount;
	}
	
	public void setBuyerId(int BuyerId)
	{
		this.BuyerId = BuyerId;
	}
	
	public void setSellerId(int SellerId)
	{
		this.SellerId = SellerId;
	}
	
	public void setReferenceNumber(String ReferenceNumber)
	{
		this.ReferenceNumber = ReferenceNumber;
	}
	
	public void setAmount(double Amount)
	{
		this.Amount = Amount;
	}
}